models/standalone-price/src/transformers.ts

```diff
@@ -19,6 +19,9 @@ const transformers = {
       'customerGroup',
       'channel',
       'tiers',
+      'custom',
+      'discounted',
+      'staged',
```

Comment:
I have no idea what's the difference between `default` and `rest`. Should we also add `replaceFields` here?